examples/api/java/SygusFun.java

```diff
@@ -125,8 +125,7 @@ public static void main(String args[]) throws CVC5ApiException
           slv.mkTerm(EQUAL, slv.mkTerm(ADD, max_x_y, min_x_y), slv.mkTerm(ADD, varX, varY)));
 
       // print solutions if available
-      if (slv.checkSynth().isUnsat())
-      {
+      if (slv.checkSynth().hasSolution()) {
```

Comment:
Format

Reply:
This is how clang-format is formatting this for me, since its Java?

Reply:
This must be something weird in your configuration, it does not for me. Can you maybe manually fix this for now?

Reply:
Fixed it for you